tests/traced_data/test_io.py

```diff
@@ -400,50 +400,30 @@ def generate_test_data():
 
         return test_data
 
-    def test_export_import(self):
-        file_path = path.join(self.test_dir, "json_test.json")
-
-        data = self.generate_test_data()
-
-        with open(file_path, "w") as f:
-            TracedDataJsonIO.export_traced_data_iterable_to_json(data, f, pretty_print=True)
-
-        with open(file_path, "r") as f:
-            imported = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
-
-        for x, y in zip(data, imported):
-            self.assertEqual(x, y)
-
-    def test_export_traced_data_iterable_to_json(self):
+    def test_export_traced_data_iterable_to_jsonl(self):
```

Comment:
Why are we removing this test?

Reply:
@lukechurch because test_round_trip does everything this does.